providers/defltprov.c

```diff
@@ -15,6 +15,12 @@
 #include <openssl/core_names.h>
 #include <openssl/params.h>
 #include "prov/implementations.h"
+#include "prov/providercommon.h"
+#include "prov/provider_util.h"
+#include "internal/nelem.h"
+
+#define ALGC(NAMES, FUNC, CHECK) { { NAMES, "default=yes", FUNC }, CHECK }
+#define ALG(NAMES, FUNC) ALGC(NAMES, FUNC, NULL)
```

Comment:
Defining and using these macros has nothing to do with moving AES_CBC_HMAC_SHA ciphers per se.  I would very much like to see this in a separate commit that tells the story.  I'm not asking for a separate PR, mind you, just a separate commit.

Reply:
separate commit added..